### ISSUE TITLE
Component Library: Uncaught (in promise) Error: addComponentAsRefTo(...)

### ISSUE THREAD

**Issue by mantovanig** [erikras/redux-form-material-ui#184]
Hi,

I'm developing a library for a react component.

If in my library i use the component TextField:
```
import { TextField } from 'redux-form-material-ui';
import { Field } from 'redux-form/immutable';

class TextInput extends React.Component {
  render() {

    const { name } = this.props;

    const classes = `textInput textInput--${name}`;

    return (
      <Wrapper className={classes}>
        <Field {...this.props} component={TextField} />
      </Wrapper>
    )
  }
}
```

When I use the library in my app I get this error:
`Uncaught (in promise) Error: addComponentAsRefTo(...)`

If I replace TextField with a simple component with an html input the error disappears.

Any suggestion?
Thanks

**Comment by mantovanig** [comment]
I found the problem.

It resolves [with this fix on 5.0](https://github.com/erikras/redux-form-material-ui/commit/e4d7d110da20f20796600dd1b6ca2ebb64b48eeb#diff-aeba1df380a617a732989815b1941d3bR19)

@erikras When will the 5.0 be released?

Thanks

**Comment by mihirsoni** [comment]
@mantovanig  It is released you can use it with
```bash
npm install --save redux-form-material-ui@next
```
